### ISSUE TITLE
[A11y] NVDA does not read out data point information while tabbing through

### ISSUE THREAD

**Issue by umkoksal** [highcharts/highcharts#13279]
#### Expected behaviour
While tabbing through the data points in a line chart, NVDA should be reading out the information related to the currently traversed data point.

#### Actual behaviour
NVDA does not read out the information related to the data point.

#### Live demo with steps to reproduce
![Bug 1455300_NVDA is not reading position and information in the tooltips](https://user-images.githubusercontent.com/61870580/78810826-62909b00-7975-11ea-9d4e-02b16521d33c.gif)

#### Product version
5.0.14

#### Affected browser(s)
Google Chrome 73.0.3683.103

**Comment by sebastianbochan** [comment]
Hi @umkoksal,
Thank you for reporting about the problem.

FYI @oysteinmoseng

**Comment by oysteinmoseng** [comment]
@umkoksal Are you able to reproduce this in a jsFiddle or Codepen for debugging? Which version of NVDA are you running?

**Comment by umkoksal** [comment]
@oysteinmoseng I have created the JSFiddle with the latest version of HighCharts and was actually unable to reproduce the problem in that case. I will go ahead and close this issue, we will start the progress of upgrading the version of our HighCharts. 

Thanks a lot for your help
